backend/controllers/unlogControllers.js

```
@@ -203,18 +203,22 @@ const logout = (req, res) => {
-    return isNaN(parsedValue) ? null : parsedValue;
-  }
-  if (type === 'date') {
-    const date = new Date(value);
-    return isNaN(date.getTime()) ? null : date;
+  switch (type) {
+    case 'string': {
+      const result = typeof value === 'string' ? value.trim() : '';
+      return String(result);
+    }
+    case 'number': {
+      const parsed = parseFloat(value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)